src/next-transpile-modules.js

```diff
@@ -53,6 +53,11 @@ const withTmInitializer = (transpileModules = []) => {
 
     const includes = generateIncludes(transpileModules);
     const excludes = generateExcludes(transpileModules);
+    const hasInclude = (ctx, req) => {
+      return includes.find((include) =>
+        req.startsWith('.') ? include.test(path.resolve(ctx, req)) : include.test(req)
```

Comment:
Can you quickly just tell me what this check is doing?

Reply:
I'm not sure either -- this is just refactoring the common parts of a function that already existed.

if I had to guess, it seems to me that this is looking for stuff inside `.next/...` and resolving those paths differently before checking whether they're in `includes`.